types/steam-tradeoffer-manager/index.d.ts

```diff
@@ -882,4 +651,12 @@ declare namespace TradeOfferManager {
         '10': string;
         '11': string;
     }
+
+    type EResultError = Error & { eresult?: TradeOfferManager.EResult; };
```

Comment:
You can (and must) omit the `TradeOfferManager` prefix as we're already in this namespace :)

Reply:
Annoying strict rules :D

Reply:
I know the struggle, but imagine the quality of this repo if there were no rules. 😄